tests/LSP1UniversalReceiver/LSP1UniversalReceiverDelegateUP.behaviour.ts

```diff
@@ -87,7 +87,7 @@ export const shouldBehaveLikeLSP1Delegate = (
     it("should support LSP1Delegate interface", async () => {
```

Comment:
Don't forget to edit the test title 🙂 

```suggestion
    it("should support LSP1 interface", async () => {
```